tutorials/haskell/servant-auth/code/servant-auth.cabal

```diff
@@ -0,0 +1,28 @@
+name:                 servant-auth
+version:              0.1.0
+cabal-version:        >= 1.10
+license:              MIT
+license-file:         LICENSE.md
+author:               Mark Karpov <markkarpov@openmailbox.org>
+maintainer:           Mark Karpov <markkarpov@openmailbox.org>
+homepage:             https://github.com/mrkkrp/servant-auth
```

Comment:
Does not exist the url... Probably is this one https://github.com/mrkkrp/servant-auth-cookie